lib/usd/pxrUsdPreviewSurface/usdPreviewSurface.cpp

```diff
@@ -399,7 +424,16 @@ MStatus PxrMayaUsdPreviewSurface::compute(const MPlug& plug, MDataBlock& dataBlo
             = depNodeFn.attribute(PxrMayaUsdPreviewSurfaceTokens->OpacityAttrName.GetText());
         const MDataHandle opacityData = dataBlock.inputValue(opacityAttr, &status);
         CHECK_MSTATUS(status);
-        const float opacity = opacityData.asFloat();
+        float opacity = opacityData.asFloat();
+
+        const MDataHandle opacityThresholdData
```

Comment:
To remove needing to make it a static member, I think we'd re-fetch `opacityThresholdAttr` by name here similar to what we're doing with `opacityAttr` just above.